Lib/test/test_import/__init__.py

```diff
@@ -1980,10 +1980,6 @@ class SinglephaseInitTests(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        if '-R' in sys.argv or '--huntrleaks' in sys.argv:
-            # https://github.com/python/cpython/issues/102251
-            raise unittest.SkipTest('unresolved refleaks (see gh-102251)')
```

Comment:
This trick cannot be used with `./python -m test -j n`, thus causing failures on CI.